apps/src/sites/studio/pages/rubrics/edit.js

```diff
@@ -1,9 +1,21 @@
 import React from 'react';
 import ReactDOM from 'react-dom';
 import RubricsContainer from '@cdo/apps/lib/levelbuilder/rubrics/RubricsContainer';
+import getScriptData from '@cdo/apps/util/getScriptData';
 
 // Note that I will need to pass some data in here (perhaps the id of the rubric)
-// Currently, this URL will work http://localhost-studio.code.org:3000/rubrics/12/edit
 $(document).ready(() => {
-  ReactDOM.render(<RubricsContainer />, document.getElementById('form'));
+  const lessonData = getScriptData('lessonData');
+  const unitName = lessonData.unit.displayName;
+  const lessonNumber = lessonData.lessonNumber;
+  const levels = lessonData.levels;
+
+  ReactDOM.render(
+    <RubricsContainer
+      unitName={unitName}
+      lessonNumber={lessonNumber}
+      levels={levels}
```

Comment:
I am hoping to add the hasRubric prop here when we have rubric data ready to go.

Reply:
Do you need the `hasRubric` prop for the rubric editor?